change/@itwin-presentation-components-3cbd3de2-872c-4136-81cc-54e6de289025.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
+  "comment": "Make \"Accept\" button to be enabled when opening the dialog with pre-filled values",
```

Comment:
This comment is going to appear in our consumer-facing changelog, so it should be as clear as possible. At the moment it would be impossible to tell what "Accept" button the comment refers to - I suggest prefixing the comment with `InstanceFilterDialog:`.

Also, I believe the change to set the editor to `StandardEditorNames.NumericInput` for all numeric properties might be a big deal to consumers. We should add another changelog entry for that. Looks like beachball doesn't support multiple change entries per file, so we might need a separate change file for it.

Reply:
fixed